cmd/apiserver/app/server/server.go

```diff
@@ -107,9 +107,9 @@ func NewCommandServer(out io.Writer) *cobra.Command {
 	return cmd
 }
 
-// runServer is a method on the options for composition. allows embedding in a
+// RunServer is a method on the options for composition. Allows embedding in a
 // higher level options as we do the etcd and serving options.
-func (serverOptions ServiceCatalogServerOptions) runServer() error {
+func (serverOptions ServiceCatalogServerOptions) RunServer(stopCh <-chan struct{}) error {
```

Comment:
You added this just so you can stop this inside test code, right? I'm fine with that. Just making sure I understand the motivation.